manifests/params.pp

```diff
@@ -88,6 +88,17 @@
       $root_group        = undef
     }
 
+    'Archlinux' : {
+      $dir               = '/etc/puppetlabs/puppet'
+      $codedir           = '/etc/puppetlabs/code'
+      $logdir            = '/var/log/puppetlabs/puppet'
+      $rundir            = '/var/run/puppetlabs'
```

Comment:
@simonsd can't we just set this to /run/puppetlabs? (/var/run should be a symlink to /run, but i'd rather see the real path here than trough a symlink.)